### PR TITLE
[OSDEV-835] API. Facility/{id} endpoint returns user emails for "activity_reports"

### PR DESCRIPTION
**Pull request by vladsha-dev**
[[OSDEV-835](https://opensupplyhub.atlassian.net/browse/OSDEV-835)] API. Facility/{id} endpoint returns user emails for "activity_reports"

1. Since the FacilityIndex model is primarily used to store cached facility data and display it publicly via the `/facilities/{id} `API endpoint, only public data can be shown. Therefore, caching emails to the FacilityIndex model was removed from the PostgreSQL indexing functions. All instances where emails are publicly displayed have been removed. The only remaining field is `ppe_contact_email`, but all functionality and code related to PPE will be deleted in this [OSDEV-562](https://opensupplyhub.atlassian.net/browse/OSDEV-562) ticket.
2. Logically structure files in the `./doc` folder. This is not necessary to mention in the RELEASE-NOTES.md file.

### CODE REVIEW

File: src/django/api/models/facility/facility.py [hunk 1]
```diff
@@ -354,19 +354,3 @@ def current_tile_cache_key():
             tile_version = 0
 
         return f'{timestamp}-{tile_version}'
-
-    def activity_reports(self):
```
Comment (VadimKovalenkoSNF) [inline src/django/api/models/facility/facility.py:358]: Could you explain why did you remove ` activity_reports` function?

Comment (roman-stolar) [reply src/django/api/models/facility/facility.py:358]: +1

File: src/django/api/migrations/0136_remove_indexing_unnecessary_emails.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from django.db import connection
+from django.db.migrations import Migration, RunPython
+
+from api.migrations._migration_helper import MigrationHelper
+
+helper = MigrationHelper(connection)
+
+
+def update_indexing_functions(apps, schema_editor):
+    helper.run_sql_files([
```
Comment (roman-stolar) [inline src/django/api/migrations/0136_remove_indexing_unnecessary_emails.py:12]: add comment what this SQLs change

**Comment by roman-stolar** [comment]
Please fix merge conflicts

File: src/django/api/models/facility/facility.py (hunk 1 above)
Comment (vladsha-dev) [reply src/django/api/models/facility/facility.py:358]: Since this function is related to activity reports I performed a small refactoring and simply removed an unused function.

**Comment by vladsha-dev** [comment]
> Please fix merge conflicts

Done.

File: src/django/api/migrations/0136_remove_indexing_unnecessary_emails.py (hunk 2 above)
Comment (vladsha-dev) [reply src/django/api/migrations/0136_remove_indexing_unnecessary_emails.py:18]: Done.

File: src/django/api/models/facility/facility.py (hunk 1 above)
Comment (VadimKovalenkoSNF) [reply src/django/api/models/facility/facility.py:358]: Ack

File: doc/release/RELEASE-NOTES.md [hunk 3]
```diff
@@ -12,9 +12,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Database changes
 #### Migrations:
 * 0135_disable_duplicates_and_lowercase_all_emails.py - implementing all emails to lowercase and disables duplicates
+* 0136_remove_indexing_unnecessary_emails.py - This migration replaces the old `index_activity_reports_info` and `index_approved_claim` functions with similar ones that do not index emails.
 
 #### Scheme changes
-* *Describe scheme changes here.*
+* [OSDEV-835](https://opensupplyhub.atlassian.net/browse/OSDEV-835) - Since the FacilityIndex model is primarily used to store cached facility data and display it publicly via the `/facilities/{id}` API endpoint, only public data can be shown. Therefore, caching emails to the FacilityIndex model was removed from the PostgreSQL indexing functions. All instances where emails are publicly displayed have been removed. The only remaining field is `ppe_contact_email`, but all functionality and code related to PPE will be deleted in this [OSDEV-562](https://opensupplyhub.atlassian.net/browse/OSDEV-562) ticket.
```
Comment (VadimKovalenkoSNF) [inline doc/release/RELEASE-NOTES.md:18]: > Therefore, caching emails to the FacilityIndex model was removed from the PostgreSQL indexing functions

Are there any other places of software where cached emails might be in use?

Comment (vladsha-dev) [reply doc/release/RELEASE-NOTES.md:18]: I verified that. There are no other places.

Comment (VadimKovalenkoSNF) [reply doc/release/RELEASE-NOTES.md:18]: Got it.

**Review by VadimKovalenkoSNF — APPROVED** [review]
Tested locally, works as expected. Approved.

**Review by roman-stolar — APPROVED** [review]
Lgtm
